src/hotspot/share/utilities/debug.cpp

```diff
@@ -511,7 +511,7 @@ extern "C" JNIEXPORT void ps() { // print stack
     f = f.sender(&reg_map);
     tty->print("(guessing starting frame id=" PTR_FORMAT " based on current fp)\n", p2i(f.id()));
     p->trace_stack_from(vframe::new_vframe(&f, &reg_map, p));
-    f.pd_ps();
+    f.print_on(tty);
```

Comment:
I agree with @shipilev - as pd_ps() was a no-op on all platforms it should simply be deleted from this code, not replaced by `f.print_on`.

Reply:
you are right -  I removed the call to `f.print_on(tty)` now. Thanks!